source/_components/sensor.fido.markdown

```diff
@@ -24,6 +24,7 @@ sensor:
   - platform: fido
     username: MYUSERNAME
     password: MYPASSWORD
+    number: MYPHONENUMBER
```

Comment:
Optional variables should not be included in the example.
Feel free to add an advanced example.